compose-v2/README.md

```diff
@@ -0,0 +1,162 @@
+# Galaxy Docker Compose
+This setup is built on the idea to use a basic docker-compose file and extend it
+for additional use cases. Therefore the `docker-compose.yml` is the base of the
+whole setup. By concatinating additional files, you can extend it to use, for
+example, HTCondor (see [Usage](#usage)).
+
+All working data (database, virtual environment, etc.) is exported in the
+`EXPORT_DIR`, which defaults to ./export.
+
```

Comment:
can we add a similar matrix as https://github.com/bgruening/docker-galaxy-stable/tree/cap_matrix#capability-matrix---toc

to the readme?